evm/src/witness/transition.rs

```diff
@@ -145,7 +145,7 @@ pub(crate) fn decode(registers: RegistersState, opcode: u8) -> Result<Operation,
         (0xf6, true) => Ok(Operation::GetContext),
         (0xf7, true) => Ok(Operation::SetContext),
         (0xf8, true) => Ok(Operation::Mload32Bytes),
-        (0xf9, true) => Ok(Operation::ExitKernel),
+        (0xf9, _) => Ok(Operation::ExitKernel),
```

Comment:
Hmm, this is insufficient, without additional changes, to make sure the opcode isn't called outside of context 0. Why has it been updated?